src/Honeybee_ExportEPC.py

```diff
@@ -127,18 +140,48 @@ def main(HBZones):
     # return surface data
     return srfData
     # done!
-
-
+#print srfData
 srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
 directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
 
+
```

Comment:
Yes, it would be awesome if you could provide some sample code. The whole point of the using the EPC is that it is computationally light. I definitely could use some help here.